### ISSUE TITLE
What is the training process of the faster rcnn in the code?

### ISSUE THREAD

**Issue by Bigwode** [chenyuntc/simple-faster-rcnn-pytorch#81]
Is the four-step alternating training method in the paper reflected in the code?
I see the code feels like the loss function was only updated once in the last step. No traces of RPN and Fast RCNN loop training were found. Can you help explain this?

**Comment by karanchawla** [comment]
Yes, you can train the RPN and ROIHead alternatively or you can combine the loss function and train the whole network together. I think the latter is a cleaner way to do it, which is what is implemented by @chenyuntc
